.classpath

```diff
@@ -13,19 +13,14 @@
 	</classpathentry>
 	<classpathentry kind="src" output="target/test-classes" path="src/test/java">
```

Comment:
I don't expect the classpath files to be present as part of the checkin. All the classpath files should be ignored while pushing into the GIT.

Reply:
Ah yes, I did wonder about that! Class files are already in .gitignore - will add the .classpath in too :D